tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,300 @@
++=========================+==========================+==========================+==========================+
+| Sub-surface scattering  | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
+|                         |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+
+Shader features
+~~~~~~~~~~~~~~~
+
+See :ref:`doc_shading_reference` for more information.
+
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Feature                 | Compatibility            | Mobile                   | Forward+                 |
++=========================+==========================+==========================+==========================+
+| Per-instance shader     | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
+| uniforms                |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Screen texture          | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Depth buffer            | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
+|                         |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Normal/Roughness buffer | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Unlit color buffer      | ❌ Not supported.*       | ❌ Not supported.*       | ❌ Not supported.*       |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Lighting buffer         | ❌ Not supported.*       | ❌ Not supported.*       | ❌ Not supported.*       |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Stencil buffer          | ❌ Not supported.        | ⚠️ Planned.              | ⚠️ Planned.              |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Custom buffer           | ❌ Not supported.        | ❌ Not supported.        | ❌ Not supported.        |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Compute shaders         | ❌ Not supported.        | ⚠️ Limited.              | ✔️ Supported.            |
```

Comment:
How do we feel about using unicode emojis here? Alternately we can use the green, yellow, and red dot images that are used in [Godot release policy](https://docs.godotengine.org/en/stable/about/release_policy.html#release-support-timeline) and a few other places.

Reply:
Fine either way, IMO.

Reply:
The emojis make more sense to use nowadays, especially since they scale better to hiDPI displays and show up directly in most text editors.

Reply:
This page also isn't the first place to use them. Emojis are used on the [minor version upgrading pages,](https://docs.godotengine.org/en/latest/tutorials/migrating/upgrading_to_godot_4.3.html#gdextension) too.